composer.json

```diff
@@ -29,7 +29,6 @@
       "homepage": "https://mvorisek.cz/"
     }
   ],
-  "version": "dev-develop",
```

Comment:
I belive this is highly needed to pass `composer install` in some situation when we are not at develop branch